corehq/blobs/tests/test_export.py

```diff
@@ -1,123 +1,89 @@
-        cls.blob_metas = []
-        cls.not_found = set()
+    def setUp(self):
+        super().setUp()
+        self.db = blob_db()
+
+    def test_only_blob_in_targeted_domain_is_exported(self):
+        # create blob that should be ignored
+        self.db.put(BytesIO(self.blob_data), meta=new_meta(domain='random', type_code=CODES.form_xml))
+        expected_meta = self.db.put(
+            BytesIO(self.blob_data), meta=new_meta(domain=self.domain, type_code=CODES.form_xml)
+        )
 
-        cls.domain_name = str(uuid.uuid4)
+        with NamedTemporaryFile() as out:
+            self.exporter.migrate(out.name, force=True)
+            with tarfile.open(out.name, 'r:gz') as tgzfile:
+                self.assertEqual([expected_meta.key], tgzfile.getnames())
 
-        for type_code in [CODES.form_xml, CODES.multimedia, CODES.data_export]:
-            for domain in (cls.domain_name, str(uuid.uuid4())):
-                meta = cls.db.put(BytesIO(data), meta=new_meta(domain=domain, type_code=type_code))
-                lost = new_meta(domain=domain, type_code=type_code, content_length=42)
-                cls.blob_metas.append(meta)
-                cls.blob_metas.append(lost)
-                lost.save()
-                cls.not_found.add(lost.key)
+    def test_different_blob_types_are_exported(self):
+        form = self.db.put(BytesIO(self.blob_data), meta=new_meta(domain=self.domain, type_code=CODES.form_xml))
+        multimedia = self.db.put(
+            BytesIO(self.blob_data), meta=new_meta(domain=self.domain, type_code=CODES.multimedia)
+        )
 
-    @classmethod
-    def tearDownClass(cls):
-        for blob in cls.blob_metas:
-            blob.delete()
-        cls.db.close()
-        super().tearDownClass()
-
-    def test_migrate_all(self):
-        expected = {
-            m.key for m in self.blob_metas
-            if m.domain == self.domain_name and m.key not in self.not_found
-        }
         with NamedTemporaryFile() as out:
-            exporter = EXPORTERS['all_blobs'](self.domain_name)
-            exporter.migrate(out.name, force=True)
+            self.exporter.migrate(out.name, force=True)
             with tarfile.open(out.name, 'r:gz') as tgzfile:
-                self.assertEqual(expected, set(tgzfile.getnames()))
-
-    def test_migrate_multimedia(self):
-        image_path = os.path.join('corehq', 'apps', 'hqwebapp', 'static', 'hqwebapp', 'images',
-                                  'commcare-hq-logo.png')
-        with open(image_path, 'rb') as f:
-            image_data = f.read()
-
-        files = (
-            (CommCareImage, self.domain_name, image_data),
-            (CommCareAudio, self.domain_name, b'fake audio'),
-            (CommCareVideo, self.domain_name, b'fake video'),
-            (CommCareAudio, 'other_domain', b'fake audio 1'),
-        )
+                self.assertCountEqual({form.key, multimedia.key}, tgzfile.getnames())
```

Comment:
nit: I don't think this test cares about the edge case where `tgzfile.getnames()` returns multiple identical names (is that even possible?), so the old way seemed simpler.
```suggestion
                self.assertEqual({form.key, multimedia.key}, set(tgzfile.getnames()))
```

Reply:
8fcdf3f0d66e09c3462a9bab8ddbe14b4454d886